src/test/OpenEXRCoreTest/general_attr.cpp

```diff
@@ -53,12 +53,12 @@ failable_malloc (size_t bytes)
     return malloc (bytes);
 }
 
-static void
-failable_free (void* p)
```

Comment:
down below we assign failable_alloc within cinit.
I'm pretty sure the intent was failable_free be correspondingly assigned within cinit as well.

Reply:
Adding the assignment to `free_fn`:
```
static exr_context_t
createDummyFile (const char* test)
{
    exr_context_t             f     = NULL;
    exr_context_initializer_t cinit = EXR_DEFAULT_CONTEXT_INITIALIZER;

    // we won't actually write to this and so don't need a proper
    // stream but need a writable context to test with.
    cinit.write_fn = dummy_write;
    cinit.alloc_fn = failable_malloc;
    cinit.free_fn = failable_free;
```
causes `OpenEXRCore.testAttrChlists` and `OpenEXRCore.testAttrLists` to fail, so something needs some further attention. I'll leave the free function in for now.

Reply:
Gotcha, can we add 

```cpp
// adding failable_malloc currently causes OpenEXRCore.testAttrChlists and OpenEXRCore.testAttrLists to fail.
// commented out until we've investigated.
//cinit.free_fn = failable_free;
```

With that I think we're good to land this